src/codeflare_sdk/cluster/cluster.py

```diff
@@ -187,6 +187,7 @@ def create_app_wrapper(self):
         write_to_file = self.config.write_to_file
         verify_tls = self.config.verify_tls
         local_queue = self.config.local_queue
+        user_labels = self.config.user_labels
```

Comment:
Perhaps `cluster_labels` would work better here if just `labels` is not an option?

Reply:
Updated to `cluster_labels` the `labels` are a local var in those functions. Or you think it's still better to call it `labels`?

Reply:
That's a good point, I think we should keep it as simple as possible for the user. Just `labels` sounds better to me, and being part of the `ClusterConfiguration` we are already implying it's cluster labels.

Reply:
Updated to be just `labels`.